server/webapp/WEB-INF/rails.new/app/assets/new_javascripts/helpers/pipeline_configs/tooltips.js.msx

```diff
@@ -232,6 +232,15 @@ define(['mithril', 'lodash', 'helpers/form_helper'], function (m, _, f) {
     </div>
   );
 
+  var materialIgnorePattern = (
+    <div>
+      <p>
+        (Optional)
+        Enter the paths to be excluded. Separate multiple entries with a comma.
```

Comment:
Change to make the message less confusing refer https://github.com/gocd/gocd/issues/2592